kuma/static/js/wiki.js

```diff
@@ -80,7 +80,7 @@
          * IF followed by [ . : ( OR capital letter
          * IF followed by 3 letters or [ @ . : (
          */
-        var split = text.split(/([a-z]{3})(?=[[.:(A-Z][@.:(A-Z]{0,}[a-zA-Z]{3})/g);
+        var split = text.split(/([a-z]{3})(?=[[.:(][@.:(A-Z]{0,}[a-z]{3})/g);
```

Comment:
If I'm understanding this change correctly, it:

1) means that the "OR capital letter" comment on line 80 is no longer correct.

2) means that CanvasRenderingContext2D will no longer get word breaks after "Canvas" and "Rendering" and for a genuinely long class name like that, this might actually cause layout problems on narrow screen sizes.

Reply:
1. Correct, I will need to update the comment.
2. I agree and was never *really* confident that this simple change was going to work without side effects.

I need to find some additional example pages to test against and consider some additional options.

> Have we tried using U+200B ZERO-WIDTH SPACE instead of ?

Do you mean instead of the `<wbr />`?

Reply:
Yes, I meant instead of `<wbr/>`. The markdown editor ate my HTML tag!